tests/conftest.py

```diff
@@ -309,18 +312,17 @@ def df_type(request: pytest.FixtureRequest):
         use_cudf = request.node.get_closest_marker("use_cudf") is not None
 
         if (use_pandas and use_cudf):
-            raise RuntimeError(
-                "Both markers (use_cpp and use_python) were added to function {}. Remove markers to support both.".
-                format(request.node.nodeid))
-        else:
-            # This will default to "cudf" or follow use_pandas
-            df_type_str = "cudf" if not use_pandas else "pandas"
+            raise RuntimeError(f"Both markers (use_cpp and use_python) were added to function {request.node.nodeid}. "
+                               "Remove markers to support both.")
+
+        # This will default to "cudf" or follow use_pandas
+        df_type_str = "cudf" if not use_pandas else "pandas"
 
     yield df_type_str
 
 
 @pytest.fixture(scope="function")
-def config(use_cpp: bool):
+def config(use_cpp: bool):  # pylint: disable=unused-argument
```

Comment:
I think its safe to disable `unused-argument` for this entire file so we dont have to repeat this everywhere